spec.md

```diff
@@ -1657,40 +1630,37 @@ If a `CreateSnapshot` operation times out before the snapshot is cut, leaving th
 2. The CO takes no further action regarding the timed out RPC, a snapshot is possibly leaked and the operator/user is expected to clean up.
 
 It is NOT REQUIRED for a controller plugin to implement the `LIST_SNAPSHOTS` capability if it supports the `CREATE_DELETE_SNAPSHOT` capability: the onus is upon the CO to take into consideration the full range of plugin capabilities before deciding how to proceed in the above scenario.
-A controller plugin COULD implement the `LIST_SNAPSHOTS` capability and call it repeatedly with the `snapshot_id` as a filter to query whether the uploading process is complete or not if it needs to upload a snapshot after it is being cut.
-
-##### Snapshot Statuses
+A controller plugin COULD implement the `LIST_SNAPSHOTS` capability and call it repeatedly with the `snapshot_id` as a filter to query whether the process is complete or not if it needs to do post processing such as uploading a snapshot after it is being cut.
 
-A snapshot could have the following statusus: UPLOADING, READY, and ERROR.
+##### The is_ready_to_use Parameter
```

Comment:
I would move this section (everything from line 1635 to 1658) up to the CreateSnapshot section of the doc. It doesn't make sense under ListSnapshot.

Reply:
Done

Reply:
So I'm moving the entire "##### The is_ready_to_use Parameter" section until line 1656 to the "#### `CreateSnapshot`" section.  I believed that is what you meant?